hack/local-up-cluster.sh

```diff
@@ -1044,6 +1044,93 @@ if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then
 
   # kubekins has a special directory for docker root
   DOCKER_ROOT="/docker-graph"
+  # enable cri 
+  # restart docker
+  # enable ipv6
+  # install cni config
+  # install cni 
+  # restart docker
+
+  echo "enable cri"
+  echo "DOCKER_OPTS=\"\${DOCKER_OPTS} --cri-containerd\"" >> /etc/default/docker
+
+  echo "restarting docker"
+  service docker restart 
+
+  echo "install cni"
+
+  CNI_DIR=${DESTDIR:=''}/opt/cni
+  CNI_CONFIG_DIR=${DESTDIR}/etc/cni/net.d
+
+  PDIR=$(pwd)
+  cd /workspace 
+  TMPROOT=$(mktemp -d)
+  git clone https://github.com/containernetworking/plugins.git "${TMPROOT}"/plugins
+  pushd "${TMPROOT}"/plugins
+  git checkout v1.0.1
+  ./build_linux.sh
+  mkdir -p $CNI_DIR
+  cp -r ./bin $CNI_DIR
+  mkdir -p $CNI_CONFIG_DIR
+  cat << EOF | tee $CNI_CONFIG_DIR/10-containerd-net.conflist
```

Comment:
we use this in kind to download the binaries and use just the ones we need

https://github.com/kubernetes-sigs/kind/blob/80a6a355a2065823bbdae5048abf76a4f2a3ec9d/images/base/Dockerfile#L171-L188

int his case, you just need to replace `ptp` by `bridge` in the script

Reply:
Thanks, this is really rough script 🙈 , will refine later once it starts passing